README.md

```diff
@@ -160,6 +160,30 @@ pandas_df = df.to_pandas()
 
 Note that the above Snowpark pandas commands will work if Snowpark is installed with the `[modin]` option, the additional `[pandas]` installation is not required.
 
+## Verifying Package Signatures
```

Comment:
Perhaps this is worth mentioning in the changelog.

Reply:
thanks for calling it out, yeah since it's related to security let's add it in the changelog